data-prepper-plugins/geoip-processor/build.gradle

```diff
@@ -25,6 +31,73 @@ dependencies {
     testImplementation project(':data-prepper-test-common')
 }
 
+task copyGeoLite2Files(type: Copy) {
```

Comment:
Rather than use a `Copy` task, you can use a `Download` task.

See this example:

https://github.com/opensearch-project/data-prepper/blob/733e7bc8381d1fe02c30b6c75c4cc2dd8984c21c/release/archives/build.gradle#L77-L81

The download plugin will help:

https://github.com/opensearch-project/data-prepper/blob/733e7bc8381d1fe02c30b6c75c4cc2dd8984c21c/release/archives/build.gradle#L7